packages/analytics-js/src/components/userSessionManager/UserSessionManager.ts

```diff
@@ -86,6 +86,10 @@ class UserSessionManager implements IUserSessionManager {
     return Boolean(sessionExpirationTimestamp && timestamp <= sessionExpirationTimestamp);
   }
 
+  getUserId(): Nullable<string> {
+    return this.storage?.get('rl_user_id') || null;
```

Comment:
we should be getting the values from the state instead. lets put a todo here for @MoumitaM to take case on the sessionmanager branch

Reply:
Addressed.